ts/Series/PieSeries.ts

```diff
@@ -432,6 +432,23 @@ seriesType<Highcharts.PieSeries>(
 
             enabled: true,
 
+            /**
+             * A
+             * [format string](https://www.highcharts.com/docs/chart-concepts/labels-and-string-formatting)
+             * for the data label. Available variables are the same as for
+             * `formatter`.
+             *
+             * @sample {highcharts} highcharts/plotoptions/series-datalabels-format/
+             *         Add a unit
+             * @sample {highmaps} maps/plotoptions/series-datalabels-format/
+             *         Formatted value in the data label
+             *
+             * @type      {string}
+             * @default   point.name
```

Comment:
I think default is set to `undefined`, but `formatter` is set to `function () { this.point.isNull ? void 0 : this.point.name; }` ? :)